docs/sdk/index.md

```diff
@@ -1,38 +1,40 @@
-<p></p>
-<p align="center"><img width="500" src="https://assets.ydata.ai/sdk/logo_SDK_col_red_black.png" alt="YData Logo"></p>
-<p></p>
-
 [![pypi](https://img.shields.io/pypi/v/ydata-sdk)](https://pypi.org/project/ydata-sdk)
 ![Pythonversion](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10-blue)
 [![downloads](https://pepy.tech/badge/ydata-sdk/month)](https://pepy.tech/project/ydata-sdk)
 
-!!! note "YData SDK for improved data quality everywhere!"
+!!! note "Fabric SDK for improved data quality everywhere!"
 
     *ydata-sdk* is here! Create a YData account so you can start using today!
```

Comment:
shouldn't we change this one from ydata-sdk to fabric-sdk? 🤔

Reply:
given the name of the package is ydata-sdk and not Fabric sdk it is best that we keep as it is, as in here is the name of the package in pypi.